src/components/v5/shared/UserInfoSectionList/partials/UserInfoList/UserInfoList.tsx

```diff
@@ -10,14 +10,14 @@ const displayName = 'v5.UserInfoSectionList.partials.UserInfoList';
 const UserInfoList: FC<UserInfoListProps> = ({ items, className }) => {
   const infoClassName = 'flex-shrink-0 text-right text-sm text-gray-900';
 
-  return items.length ? (
+  return items.length > 0 ? (
```

Comment:
I got into habit of using it like this because of stuff like `items.length && <Component />` will render `0` if the array is empty.

But I'm fine either way.

Reply:
Yeah I think we should probably avoid using && in this context because there are many ways this could go wrong. I've corrected these a couple of times already.